docs: DOC-220: Remove FF references from docs

caitlinwheeless opened the pull request:
Removing feature flag references from docs. 



#### Affects

- [X] Community docs
- [X] Enterprise docs

caitlinwheeless committed:
docs: DOC-220: Remove FF references from docs

caitlinwheeless committed:
Removing additional FF references from tag pages

caitlinwheeless committed:
Update magic wand page

luarmr commented on docs/source/tags/magicwand.md:
```diff
@@ -1,9 +1,9 @@
 order: 413
 meta_title: Magic Wand Tag for Quick Thresholded Flood Filling During Image Segmentation
 meta_description: Customize Label Studio with a Magic Wand tag to quickly click and drag to threshold flood fill image areas during image segmentation labeling for machine learning and data science projects.
+hide_sidebar: true
```
Not sure if need it, just pointing it out because looks odd in this set of changes

caitlinwheeless replied:
From my understanding, the entire Magic Wand tool is behind a FF. Rather than deleting a bunch of content from the code, I figured it would be cleaner to just hide the docs page.

caitlinwheeless replied:
Update - Put the Magic Wand page back in after discussion with Andrew.

luarmr reviewed (approved):
LGTM

hlomzik reviewed (changes requested):
Hmmm, I think it's very odd to just remove all parameters somehow connected to feature flags. The main idea of feature flags is that they will be enabled at some point (as soon as possible actually). So most of this params most probably are available for everyone and we really need them in docs. Let's discuss this.

caitlinwheeless committed:
Bringing back magic wand page but removing FF reference

caitlinwheeless committed:
Adding back param definitions

caitlinwheeless committed:
Removing empty lines

caitlinwheeless committed:
Update MagicWand.js to remove extra line

caitlinwheeless committed:
Update Number.jsx to fix missing word

caitlinwheeless committed:
Update Taxonomy.jsx

farioas commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/9505168641)
> Successfully merged:  12 files changed, 68 insertions(+), 29 deletions(-)

farioas committed:
Merge branch 'develop' into 'docs/ff-removal'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/9505168641

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/9505189292
